contrib/babelfishpg_tsql/src/tablecmds.c

```diff
@@ -156,12 +154,10 @@ lookup_and_drop_triggers(ObjectAccessType access, Oid classId,
 
 		if (pg_trigger->tgrelid == relOid && !pg_trigger->tgisinternal)
 		{
-			trigRelation = RelationIdGetRelation(relOid);
-			trig_physical_schema = get_namespace_name(get_rel_namespace(pg_trigger->tgrelid));
-			trigobjlist = list_make2(makeString(trig_physical_schema), makeString(NameStr(pg_trigger->tgname)));
-			trigAddress = (*get_trigger_object_address_hook) (trigobjlist, &trigRelation, true, false);
```

Comment:
Can we remove this hook implementation from Babelfish?

Reply:
No we can't , it's refer from other places

Reply:
It's just redundant to call from here